src/System.Management.Automation/engine/SessionState.cs

```diff
@@ -391,11 +391,6 @@ internal void SetConsoleVariable()
         {
```

Comment:
Consider removing this whole function and the special variable now that console files went away.

Reply:
will remove